guides/source/active_job_basics.md

```diff
@@ -747,24 +747,24 @@ jobs as arguments (note that this is different from `perform_later`).
 `perform_all_later` does call `perform` under the hood. The arguments passed to
 `new` will be passed on to `perform` when it's eventually called.
 
-Here is an example calling `perform_all_later` with `GuestCleanupJob` instances:
+Here is an example calling `perform_all_later` with `GuestsCleanupJob` instances:
 
 ```ruby
 # Create jobs to pass to `perform_all_later`.
 # The arguments to `new` are passed on to `perform`
-guest_cleanup_jobs = Guest.all.map { |guest| GuestsCleanupJob.new(guest) }
+cleanup_jobs = Guest.all.map { |guest| GuestsCleanupJob.new(guest) }
 
-# Will enqueue a separate job for each instance of `GuestCleanupJob`
-ActiveJob.perform_all_later(guest_cleanup_jobs)
+# Will enqueue a separate job for each instance of `GuestsCleanupJob`
+ActiveJob.perform_all_later(cleanup_jobs)
 
 # Can also use `set` method to configure options before bulk enqueuing jobs.
-guest_cleanup_jobs = Guest.all.map { |guest| GuestsCleanupJob.new(guest).set(wait: 1.day) }
+cleanup_jobs = Guest.all.map { |guest| GuestsCleanupJob.new(guest).set(wait: 1.day) }
 
-ActiveJob.perform_all_later(guest_cleanup_jobs)
+ActiveJob.perform_all_later(cleanup_jobs)
 ```
 
 `perform_all_later` logs the number of jobs successfully enqueued, for example
-if `Guest.all.map` above resulted in 3 `guest_cleanup_jobs`, it would log
+if `Guest.all.map` above resulted in 3 `guests_cleanup_jobs`, it would log
```

Comment:
This needs to be changed to `cleanup_jobs` too?

Reply:
Changed it, thanks!